test/testinput/bufr_test.yaml

```diff
@@ -0,0 +1,77 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./Data/gdas.t12z.1bmhs.tm00.bufr_d"
+
+      mnemonicSets:
+        - mnemonics: [SAID, FOVN, YEAR, MNTH, DAYS, HOUR, MINU, SECO, CLAT, CLON, CLATH, CLONH, HOLS]
+        - mnemonics: [SAZA, SOZA, BEARAZ, SOLAZI]
+        - mnemonics: [TMBR]
+          channels : 1-15
+
+      exports:
+        timestamp:
+          datetime:
+            year: YEAR
+            month: MNTH
+            day: DAYS
+            hour: HOUR
+            minute: MINU
+            second: SECO
+            isUTC: true
+
+        longitude:
+          mnemonic: CLON
+          transforms:
+            - offset: 0
+
+        latitude:
+          mnemonic: CLAT
+
+        radiance:
+          mnemonic: TMBR
+
+    ioda:
+      backend: netcdf
+      obsdataout: "./testoutput/ioda_encoder_result.nc"
```

Comment:
I think here this should be testrun not testoutput, testoutput is where the reference files go to compare to what is produced in testrun, but confirm this with some of the python tests in test/CMakeLists.txt

Reply:
Also, forgot, this bufr_test.yaml file needs added to the list of .yaml files in the test/CMakeLists.txt file